README.md

```diff
@@ -215,7 +215,7 @@ To run a scan and export operation using the CLI from docker:
 The commands are as follows:
 
 ```
-CMD_LINE_ARGS="scan mysolacebroker /path/to/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"
+export CMD_LINE_ARGS="scan mysolacebroker /path/to/scan.zip  --springdoc.api-docs.enabled=false --spring.main.web-application-type=none"
```

Comment:
While updating the file, I noticed the instructions between the UI and README were different, so I made them match.